zerver/tests/test_subs.py

```diff
@@ -3054,7 +3054,7 @@ def test_realm_admin_remove_others_from_subbed_private_stream(self) -> None:
         are on.
         """
         result = self.attempt_unsubscribe_of_principal(
-            query_count=19,
+            query_count=21,
```

Comment:
To be ignored for now: https://chat.zulip.org/#narrow/channel/3-backend/topic/stream_to_dict.20settings_group_dict.20fetching.2E